src/commands/NewIntegrationFileCommand.ts

```diff
@@ -47,7 +47,9 @@ export async function create() : Promise<void> {
 			if (filename) {
 				const kamelExe = kamel.create();
 				const newFileFullPath: string = computeFullpath(language, workspaceFolder, filename);
-				kamelExe.invoke(`init "${newFileFullPath}"`);
+				await kamelExe.invoke(`init "${newFileFullPath}"`);
```

Comment:
Would it be better to do `await kamelExe.invoke(...).then() here?

Reply:
No. invoke returns a Promise I guess. Await does resolve the result of that promise. I would not use .then afterwards.

Reply:
ok. just a thought. :)